sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/TracerProviderConfiguration.java

```diff
@@ -70,9 +69,10 @@ static List<SpanProcessor> configureSpanProcessors(
       ConfigProperties config,
       Map<String, SpanExporter> exportersByName,
       MeterProvider meterProvider,
-      List<Closeable> closeables) {
+      List<Closeable> closeables,
+      SpiHelper spiHelper) {
     Map<String, SpanExporter> exportersByNameCopy = new HashMap<>(exportersByName);
-    List<SpanProcessor> spanProcessors = new ArrayList<>();
+    List<SpanProcessor> spanProcessors = spiHelper.load(SpanProcessor.class);
```

Comment:
Hmm.. can we punt on this for now? Reasons:
- If we do want span processors to be added, there needs to an order.
- I don't see the equivalent for LogRecordProcessors or MetricReaders so it appears to be inconsistently implemented. 
- Better to let the user provide an AutoConfigurationCustomizer if they want to customize the tracer provider with additional span processors.

Reply:
Right, let's have that discussion later.